testcases/OpTestFadumpSmt.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env python2
+# IBM_PROLOG_BEGIN_TAG
+# This is an automatically generated prolog.
+#
+# $Source: op-test-framework/testcases/OpTestFadumpSmt.py $
+#
+# OpenPOWER Automated Test Project
+#
+# Contributors Listed Below - COPYRIGHT 2019
+# [+] International Business Machines Corp.
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+# IBM_PROLOG_END_TAG
+
+'''
+OpTestFadump
+------------
+
+This module can contain testcases related to FAdump.
+
+Please install crash and kernel-default-debuginfo packages on the target machine where kdump is tested.
+
+1. Configure FAdump.
+2. Trigger crash.
+3. Check for vmcore and dmesg.txt.
+4. Run crash tool on the vmcore captured.
+5. Repeat Step2 to Step4 for different SMT levels.
+'''
+
```

Comment:
Pavithra, 

We already have `OpTestKdumpLparSmt.py`. Why not intigrate this to same file. Also looks like you are duplicating code between files. Why not just have single file covering various dump scenario's including fadump, kdump etc?

-Vasant